src/components/10-atoms/input-text/README.md

```diff
@@ -146,6 +146,17 @@ The string-valued `info` provides the label text as HTML.
 
 _Note: The application is responsible for sanitizing HTML!_
 
+### pattern
+
+It is a string value. But just input a valid RegEx. Useful to open up numeric keyboard on touch devices. See official HTML documentation for details.
```

Comment:
I would write:
`It is a string value. But just input a valid RegEx. It checks the element's value on form submission. See official HTML documentation for details.`

In my opinion to open the numeric keyboard the semantic right way is to use inputmode. Therefore I wouldn't write this.